modules/twinklewarn.js

```diff
@@ -1644,16 +1475,18 @@ Twinkle.warn.callbacks = {
 		text += "|sig=true}}";
 		return text;
 	},
+	// NOTE: used by block module
 	preview: function(form) {
-		var templatename = form.sub_group.value;
-		var linkedarticle = form.article.value;
-		var templatetext;
-
-		if (templatename in Twinkle.warn.messages.block) {
-			templatetext = Twinkle.warn.callbacks.getBlockNoticeWikitext(templatename, linkedarticle, form.block_timer.value,
-				form.block_reason.value, Twinkle.warn.messages.block[templatename].indefinite);
+		var templatename = form.sub_group.value,
+				linkedarticle = form.article.value,
```

Comment:
When I make my next push the tabs will fix themselves. The tab settings in my text editor made them look the same, anyway I'm converting all leading spaces to tabs as it appears tabs is the convention with the Twinkle codebase.